.evergreen/legacy_config_generator/evergreen_config_lib/tasks.py

```diff
@@ -725,12 +704,6 @@ def pre_commands(self) -> Iterable[Value]:
 all_tasks = chain(
     all_tasks,
     [
-        PostCompileTask(
-            "test-asan-memcheck-mock-server",
-            tags=["test-asan"],
-            get_build="debug-compile-asan-clang",
-            commands=[func("run mock server tests", ASAN="on", SSL="ssl")],
```

Comment:
Remove `"run mock server tests"` function in legacy config generator `functions.py`.